.gitignore

```diff
@@ -15,7 +15,7 @@ Thumbs.db
 # Project files
 /node_modules
 /build
-/config.js
+/config.json
```

Comment:
This should go away. If there is a `config.json` it should be here `public/config.json`

Reply:
lets ignore it in both dirs